auction9-frontend/src/app/data-table/data-table.component.ts

```diff
@@ -28,23 +29,22 @@ const NAMES: string[] = [
 
 export class DataTableComponent implements AfterViewInit {
 
-  displayedColumns: string[] = ['id', 'name', 'price', 'action'];
-  dataSource: MatTableDataSource<BidData>;
+  displayedColumns: string[] = ['auctionID', 'title', 'price', 'action'];
+  dataSource: MatTableDataSource<AuctionData>;
 
   @ViewChild(MatPaginator) paginator: MatPaginator;
   @ViewChild(MatSort) sort: MatSort;
 
-  constructor() {
-    // Create 50 Bids
-    const bids = Array.from({length: 50}, (_, k) => createNewUser(k + 1));
-
-    // Assign the data to the data source for the table to render
-    this.dataSource = new MatTableDataSource(bids);
-  }
+  constructor(private auctionFetchService: AuctionFetchService) { }
 
   ngAfterViewInit() {
-    this.dataSource.paginator = this.paginator;
-    this.dataSource.sort = this.sort;
+    this.auctionFetchService.getActiveAuctions().then((data: []) => {
```

Comment:
Does this mean that `data table component` will always fetch auctions, so it can not be customized component?
Reusability of this component should be achieved.

Reply:
In new commit changed that data-table will fetch different 'data' depending on current URL